contrail-ui/src/utils/fileReader.ts

```diff
@@ -0,0 +1,16 @@
+export default (file: Blob): Promise<string | ArrayBuffer | null> => {
```

Comment:
could you explain what does this entire fn do?

Reply:
Since in the axios request I specified the response type as blob, if an error occurs it won't be able to read it because axios thinks the response is a blob instead of a JSON object. This basically reads the blob and makes it JSON-parsable so we can get the error response from the server.